### DISCUSSION TITLE
Tooltip can't wrap an Icon component.

### DISCUSSION THREAD

**Discussion by zhoudalang** [chakra-ui/chakra-ui#7828] [Q&A]
**Description:** 
When I wrapped an Icon component with Tooptip, i can't get tooltip when hovering it.
**Link to Reproduction**
https://codesandbox.io/s/stupefied-sid-g227qk?file=/App.tsx

I've tried using **forwardRef**, but it didn't work.

**Comment by HaleyWardo** [comment]
It looks like you are missing the label for the tooltip. 

```
<Tooltip label="My tooltip label">
  <CustomIcon/>
</Tooltip>
```

**Reply by zhoudalang** [reply]
Sorry to miss the label, lol.
Actually, problem was sovled by using `shouldWrapChildren`.
Thank you so much.
